tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md

```diff
@@ -111,6 +141,170 @@ you already have an app to deploy, you can use it instead.
+    Open `lib/hello_web/controllers/page_controller.ex` and rewrite
+    the `index` function as follows:
+
+        def index(conn, _params) do
+          value = Hello.Repo.aggregate(Hello.User, :count, :id)
+          conn
+          |> assign(:value, value)
+          |> render("index.html")
+        end
+
+1.  Display the value by inserting the following into the
+    `lib/hello_web/templates/page/index.html.eex` template:
+
+        <p>Number of users: <%= @value %></p>
+
+1.  Recompile and run the app
+
+        mix phx.server
+
+1.  Visit [http://localhost:4000](http://localhost:4000) to verify that your
+    new code is running. You can log into your database and add new rows, and
+    reload the page to verify that the count has changed.
+
+For more information on using Ecto to access a SQL database, see the
+[Phoenix Ecto guide](https://hexdocs.pm/phoenix/ecto.html).
+
+## Create a production database in Cloud SQL
+
+In this section, you will create your "production" database using Google Cloud
+SQL, a fully-managed database service providing PostgreSQL and MySQL in the
+cloud. If you already have a database hosted elsewhere, you may skip this
+section, but you may need to ensure your production configuration is set up to
+connect to your database.
+
+Before you begin this section, make sure you have enabled billing and the
+needed APIs in your cloud project. You should also set the default project for
+your gcloud SDK if you have not already done so:
+
+    gcloud config set project ${PROJECT_ID}
+
+### Create a Cloud SQL instance
+
+First you will create a new database in the cloud.
+
+1.  Create a Cloud SQL instance named `hellodb` with a Postgres database
+    by running the following command:
+
+        gcloud sql instances create hellodb --region=us-central \
+          --database-version=POSTGRES_9_6 --tier=db-g1-small
+
+    You may choose a region other than `us-central` if there is one closer to
+    your location.
+
+1.  Get the _connection name_ for your Cloud SQL instance by running the
+    following command:
+
+        gcloud sql instances describe hellodb
+
+    In the output, look for the connection name in the `connectionName` field.
+    The connection name has this format: `[PROJECT-ID]:[COMPUTE-ZONE]:hellodb`
+    We will refer to the connection name as `[CONNECTION-NAME]` throughout this
+    tutorial.
+
+1.  Secure your new database instance by setting a password on the default
+    postgres user:
+
+        gcloud sql users set-password postgres \
+          --instance=hellodb --prompt-for-password
+
+    When prompted, enter a password for the database.
+
+### Connect to your Cloud SQL instance
+
+In this section you will learn how to connect to your Cloud SQL instance from
+your local workstation. Generally, you will not need to do this often, but it
+is useful for the initial creation and migration of your database, as well as
+for _ad hoc_ database connection for maintenance.
```

Comment:
for _ad hoc_ database connection for maintenance  -> for creating _ad hoc_ database connections for maintenance